build/Cortex-M3_MPS2_QEMU_GCC/coremqtt-agent.mk

```diff
@@ -0,0 +1,22 @@
+#Intended to be included from the Makefile.  Builds the MQTT agent files along
+#with the appropriate interface, as well as the coreMQTT library itself.
+
+
+#Core files
+COREMQTT_AGENT_DIR += ./../../lib/FreeRTOS/coreMQTT-Agent/source
+COREMQTT_DIR = $(COREMQTT_AGENT_DIR)/dependency/coreMQTT/source
+
+INCLUDE_DIRS += -I$(COREMQTT_AGENT_DIR)/include \
+				-I$(COREMQTT_DIR)/include \
+				-I$(COREMQTT_DIR)/interface
+
+VPATH += $(COREMQTT_AGENT_DIR) $(COREMQTT_AGENT_DIR)/portable/freertos $(COREMQTT_DIR)
+SOURCE_FILES += $(wildcard $(COREMQTT_AGENT_DIR)/*.c)
+SOURCE_FILES += $(wildcard $(COREMQTT_AGENT_DIR)/portable/freertos/*.c)
```

Comment:
I do not believe the version of the coreMQTT agent submodule that is in this repository has this directory any more. It was moved to `mqtt-agent-interface/` which you already add below.

Reply:
Good catch - corrected.